### ISSUE TITLE
Pull Date Added from Navidrome

### ISSUE THREAD

**Issue by isle9** [evilmanimani/beets-navidrome_sync#2]
Hey, are there any plans to add this? I can't really use modified dates from files since they are a bit wrong unfortunately (and there seems to be now way to use creation date on Linux?).

**Comment by evilmanimani** [comment]
Yeah that was my understanding, I'll have to look into it some more, if you happen to figure anything out in the meantime though let me know.
